test/generate.test.js

```diff
@@ -126,6 +126,28 @@ test('generateImage', function(t) {
     t.end();
 });
 
+// Generating both a valid layout and image in one pass
+test('generateImage with format:true', function(t) {
+    [1, 2, 4].forEach(function(scale) {
+        t.test('@' + scale, function(tt) {
+            var optimizedPngPath = path.resolve(path.join(__dirname, 'fixture/sprite@' + scale + '-64colors.png'));
+            spritezero.generateLayout({ imgs: getFixtures(), pixelRatio: scale, format: true }, function(err, formatted, formatted2) {
```

Comment:
Nit: Maybe consider renaming `formatted` and `formatted2`?

Reply:
done in 7ee01d4